CONTRIBUTING.rst

```diff
@@ -80,10 +79,9 @@ Ready to contribute? Here's how to set up `git_wrapper` for local development.
    tests, including testing other Python versions with tox::
 
     $ flake8 git_wrapper tests
-    $ python setup.py test or py.test
+    $ py.test tests
```

Comment:
If you want to call pytest directly, you can do so simply with 'pytest' and it will find the standard directories.  Alternatively, you can have people run 'make test' for their current pipenv, or 'make test-all' to run tox against all supported versions